src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/appsettings.Development.json

```diff
@@ -7,7 +7,7 @@
     }
   },
   "UseStubProviderRelationships": true,
-  "UseStubProviderAuth": true,
+  "UseStubProviderAuth": false,
```

Comment:
should we be checking in changes to these settings?

Reply:
some one chekced in true few weeks ago, reverting it to way it was before.